code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -101,12 +101,37 @@
 
 /obj/item/weapon/implant/explosive/activate(var/cause)
 	if(!cause || !imp_in)	return 0
-	if(cause == "action_button" && alert(imp_in, "Are you sure you want to activate your explosive implant? This will cause you to explode and gib!", "Explosive Implant Confirmation", "Yes", "No") != "Yes")
+	if(cause == "action_button" && alert(imp_in, "Are you sure you want to activate your microbomb implant? This will cause you to explode!", "Microbomb Implant Confirmation", "Yes", "No") != "Yes")
 		return 0
-	explosion(src,0,1,5,7,10, flame_range = 5)
-	if(imp_in)
+	var/light = 2
+	var/medium = 1
+	var/heavy = 0.5
+	for(var/obj/item/weapon/implant/explosive/E in imp_in)
+		heavy += 0.5
+		medium += 1
+		light += 2

```

Comment:
Wait, why would you buy the macrobomb implant at all? You hit maxcap with 10 of these, all that increases past that is flame range and that's not really worth it.